src-docs/src/views/guidelines/colors/_utilities.js

```diff
@@ -151,8 +152,8 @@ color: $${foreground};`;
             onClickAriaLabel="Click to copy SASS configurations"
             disabled={!contastIsAcceptableToCopy}
             style={{
-              backgroundColor: palette[background].rgba,
-              color: palette[foreground].rgba,
+              backgroundColor: palette[background],
+              color: palette[foreground],
```

Comment:
@Willie-The-Lord, should not be here rgba values? Although I think is not a big deal, because the result is the same. I suppose that that property .rgba was a string in the format "rgba(<0-255>,<0-255>,<0-255>,<0-1>).

Just saying to keep consistency with what there was before. However, I don't see any utility function that returns the equivalent format for RGBA, just one for converting a HEX to RGB, but in array format.

Reply:
Hi @BigSamu, I've just re-run the original code and `palette[background].rgba` is causing the render error as I mentioned in previous comments. This is because `palette[background]` returns a `string`, which does not have an `.rgba` property.

#### ❌ `palette[background].rgba`
![Screenshot 2023-10-25 at 1 16 29 PM](https://github.com/opensearch-project/oui/assets/65143821/d23b28b4-30bf-4939-a1bd-7705dc761196)

#### ✅ `palette[background]`
![Screenshot 2023-10-25 at 1 15 55 PM](https://github.com/opensearch-project/oui/assets/65143821/a4c520d9-55f3-474b-91c3-35ec3e64881c)

Reply:
I know! What I am trying to say is that the code was using RGBA format before and maybe for something (I don't know). That is why I suggest sticking with that format instead of HEX format. Make sense? However, you will need to build utility functions for getting RGBA values. It is just for consistency. 

Of course, It is a really minor issue. Your solution is much more simple and straightforward.

Reply:
Ah, I understand now. I can also take on the task of building the utility function. @BSFishy , do you have any thoughts or suggestions on this?